apps/settings/components/form_card/index.coffee

```diff
@@ -12,15 +12,22 @@ module.exports = ($el) ->
 
     serializer = new Serializer $el
 
+    onSubmit = if onSubmit then onSubmit else ->
+      promise = new Promise
+      promise.resolve()
```

Comment:
I mean I see it here obviously but I don't understand what it could be doing.

Reply:
https://github.com/aredotna/ervell/pull/1246/files#diff-5a9d3936c693b3f81620b002aa5b99c5R14

Reply:
`Promise.resolve`